src/types/Enums.ts

```diff
@@ -48,3 +48,8 @@ export enum CountryCode {
 export enum DiscountCodeType {
   COUPON,
 }
+
+export enum PlatFormTitle {
+  AMAZON,
+  NYKAA,
```

Comment:
```suggestion
  NYKAA
```

Reply:
ye prettier se aa raha hai

Reply:
hamne discuss kiya tha ki abhi jo prettier kar raha hai karne do, jab eslint and prettier sab repo se setup karenge tab ek default structure bana denge sab ke liye